astropy/time/formats.py

```diff
@@ -710,83 +719,48 @@ class TimeUnix(TimeFromEpoch):
-      times increases linearly through the day by 1.0.  See also the
-      documentation for the `~astropy.time.TimeUnix` class.
-
-    Examples
-    --------
-
-      >>> from astropy.time import Time
-      >>> t = Time('2020-01-01', scale='utc')
-      >>> t.unix_tai - t.unix
-      29.0
-
-      >>> t = Time('1970-01-01', scale='utc')
-      >>> t.unix_tai - t.unix  # doctest: +FLOAT_CMP
-      8.200000198854696e-05
-    """
-    name = 'unix_tai'
-    epoch_val = '1970-01-01 00:00:08'
-    epoch_scale = 'tai'
+    Unix time (TAI): SI seconds elapsed since 1970-01-01 00:00:00 TAI (see caveats).
 
-
-class TimeUnixPTP(TimeUnix):
-    """
-    Precision Time Protocol: SI seconds elapsed from 1970-01-01 00:00:00 TAI.
+    This will generally differ from standard (UTC) Unix time by the cumulative
+    integral number of leap seconds introduced into UTC since 1970-01-01 UTC.
 
     This convention matches the definition of the Precision Time Protocol,
-    https://en.wikipedia.org/wiki/Precision_Time_Protocol and
+    https://en.wikipedia.org/wiki/Precision_Time_Protocol, which
     is also used by the White Rabbit protocol in High Energy Physics:
     https://white-rabbit.web.cern.ch/
+    and the linux CLOCK_TAI (https://www.cl.cam.ac.uk/~mgk25/posix-clocks.html).
 
     For more information on this time format definition, see:
     https://www-acc.gsi.de/wiki/Timing/TimingSystemButisInterface#Time
 
-    This format is basically the same as the ``unix_tai`` with an offset of
-    8 seconds due to the different epoch (1970-01-01 00:00:00 UTC vs. TAI).
-
-    This will generally differ from Unix time by the cumulative integral number
-    of leap seconds since 1970-01-01 UTC plus the initial offset of 8 seconds.
-
     Caveats:
+
     - Before 1972, fractional adjustments to UTC were made, so the difference
-      between ``unix`` and ``unix_ptp`` time is no longer an integer.
-    - Because of the fractional adjustments, to be very precise, ``unix_ptp``
+      between ``unix`` and ``unix_tai`` time is no longer an integer.
+    - Because of the fractional adjustments, to be very precise, ``unix_tai``
       is the number of seconds since ``1970-01-01 00:00:00 TAI`` or equivalently
-      ``1969-12-31 23:59:51.999918 UTC``.  The difference between TAI and UTC
+      ``1969-12-31T23:59:51.999918 UTC``.  The difference between TAI and UTC
       at that epoch was 8.000082 sec.
-    - On the day of a leap second the difference between ``unix`` and ``unix_ptp``
-      times increases linearly through the day by 1.0.  See also the
+    - On the day of a positive leap second the difference between ``unix`` and
+      ``unix_tai`` times increases linearly through the day by 1.0. See also the
       documentation for the `~astropy.time.TimeUnix` class.
+    - While no negative leap seconds were yet introduced into UTC, these are
+      possible in the future, if the length of the astronomical day further
```

Comment:
This is difficult, as the length of the day increases due to tides of the moon (the poles melting does not help...), though perhaps a fluctuation can cause it...

Reply:
Now I am a bit confused, I remember reading an announcment by the IERS that the day shortened so much in 2020 that there are not any positive leap seconds on the horizon anymore. I will reword to make clear that negative leap seconds are unlikely.

Reply:
https://www.timeanddate.com/time/earth-faster-rotation.html

Reply:
Consider a more neutral statement, such as "Negative leap seconds are also possible, though none have been needed as of 2020-01".

I did not know that the day length has been decreasing and it's very interesting, but it seems out of scope for this documentation.

Reply:
I find it important to stress that there *might* be negative leap seconds

Reply:
Ha, I had missed that completely!  Short-term trends can clearly overwhelm the long-term behaviour....  Anyway, agreed, do keep a statement, but just keep it a bit more general like suggested by @r-owen (the long-term trend better be for days getting longer!).